cmd/support-cron-scheduler/res/db/sql/01-tables.sql

```diff
@@ -0,0 +1,24 @@
+--
+-- Copyright (C) 2024 IOTech Ltd
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+-- cron_scheduler.schedule_job is used to store the schedule job information
+CREATE TABLE IF NOT EXISTS cron_scheduler.schedule_job (
+    id UUID PRIMARY KEY,
+    name TEXT NOT NULL,
+    job JSONB NOT NULL,
```

Comment:
```suggestion
    content JSONB NOT NULL,
```

use a general field name `content`